upi/azure/install-config.yaml.example

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
I don't think should be commiting another install-config.yaml 
the installer generates one for users `openshift-install create install-config`

Reply:
Done.